### DISCUSSION TITLE
Error: selenium-manager.exe  --browser "chrome" --language-binding csharp --output json

### DISCUSSION THREAD

**Discussion by Napoleon10** [reqnroll/Reqnroll#232] [Q&A] [answered]
Hello all, 
I have been searching and waiting (for a fix) to no avail yet! There are some similar issues that have arisen previously in Github and google community forums etc; around Selenium Manager with the above error message -this includes the Edge browser too. It was all working around 6 weeks ago; could this be an ReqNroll issue? Issue occurs when you start to Run or Debug tests in the Test Explorer of Visual Studio! 

This doesn't seem to be an universal issue - otherwise everyone would be 'jumping up and down'! 
Any clues any one?

I am using:
Visual Studio 2022
ReqNroll extension
MSTest Framework
Latest Edge or Chrome Browser
Windows 10 Machine.

**Comment by gasparnagy** [comment, marked as answer]
Could you please explain a bit more in detail what the problem is and what the error message you see?

You can try to reproduce the error with our sample project as well: https://github.com/reqnroll/Sample-ReqOverflow/

**Reply by Napoleon10** [reply]
I have re-created a new project and this is not producing the above error. Thanks for your time Gaspar. I will mark this as answered, ta.

**Reply by gasparnagy** [reply]
no prob

**Comment by Napoleon10** [comment]
Thank you for your response and sorry for the time taken to reply.

I have used your sample project successfully on my own computer. This will most likely be a work related issue - not sure what though?!
As it was working previously at my workplace. I will investigate further tomorrow with your sample project at my work station.


Regards,


John.


________________________________
